interview-questions.md

```diff
@@ -7,52 +7,52 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What are props in React?
 
-  Your answer:
+  Your answer: Props in React is short for properties. Props are used to pass data from the child component to the parent component.
 
-  Researched answer:
+  Researched answer: Props is a keyword in React that stands for properties. In a nutshell, props are used tp pass data from the parent component to a child component and is largely responsible for allowing a react component pattern to work.
 
 
 
 2. What is a DOM event?
 
-  Your answer:
+  Your answer: An event is considered to be any changes being made by the user to the DOM. For example, a user reloading a webpage is considered a DOM event.
 
-  Researched answer:
+  Researched answer: An event is considered to be any changes being made by the user to the DOM. For example, a user reloading a webpage is considered a DOM event.
 
 
 
 3. What is object-oriented programming? How is it different than functional programming?
 
-  Your answer:
+  Your answer: Object-oriented programming is focused on a concept centered around objects. It's different from functional programming because functional programming emphasizes on evaluation of functions.
 
-  Researched answer:
+  Researched answer: Object-oriented programming is focused on a concept centered around objects. It's different from functional programming in a sense that objects represent things you are programming about. The objects hold data about them in attributes. Whereas, the attributes in the objects are altered through methods or functions that are given to the object using functional programming.
```

Comment:
Good work up here! I think diving a little deeper on these topics will be really helpful for understanding them.